crates/l3/appchain-utils/src/lib.rs

```diff
@@ -18,7 +20,12 @@ pub type TransactionExecution<'a> = Execution<'a, RpcAccount<'a>>;
 
 pub const NO_CONSTRUCTOR_ARG: Vec<FieldElement> = Vec::new();
 
-pub const MAX_FEE: &str = "0x1000000000000";
+pub const MAX_FEE: FieldElement = FieldElement::from_mont([
```

Comment:
Can you add a comment explaining which value it is?
```rust
/// 0x1000000000000
```

Reply:
done